ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -28,3 +28,7 @@ tests:
   commands: dist/pr_check.sh
```

Comment:
let's rename this script to `cargo_test`

Reply:
done, also https://github.com/openshift/cincinnati/pull/110